Source/santad/Logs/EndpointSecurity/Logger.mm

```diff
@@ -97,4 +97,9 @@
   writer_->Write(serializer_->SerializeDiskDisappeared(props));
 }
 
+void Logger::LogAccess(const Message &msg) {
+  // TODO(xyz): Should likely take an EnrichedMessage
+  LOGE(@"GOT ACCESS EVENT");
```

Comment:
Definitely a place holder. An upcoming PR will be filling out the proto serializer for this new event. Didn't bother leaving in my debug logs for now since I left the client disabled for this PR.